app/projects/current/page.tsx

```
@@ -0,0 +1,56 @@
+"use client";
+
+import { DotLottieReact } from "@lottiefiles/dotlottie-react";
```

ESLint

[error] 'DotLottieReact' is defined but never used.

(@typescript-eslint/no-unused-vars)